src/common/dsp/vembertech/basic_dsp.cpp

```diff
@@ -262,7 +262,8 @@ float sine_ss(unsigned int x) // using 24-bit range as [0..2PI] input
 
     return P * (y * abs(y) - y) + y;   // Q * y + P * y * abs(y)   */
 }
-#if !_M_X64 && !defined(ARM_NEON)
+
+#if 0
```

Comment:
Do you mind just deleting this entire block rather than if 0ing it?

Reply:
done

Reply:
Great
When ci finishes I will merge
Thanks for upstreaming